modules/database/repository.py

```
@@ -0,0 +1,111 @@
+        :type library_names: list[str]
+        :return: A list of "recently added" items in the past x minutes for specific libraries
+        """
+        try:
+            database_entries: list[RecentlyAddedItemDatabaseModel] = (
+                self._database.get_all_recently_added_items_in_past_x_minutes_for_libraries(
+                    minutes=minutes,
+                    library_names=library_names)
+            )
+            return [RecentlyAddedItemModel.from_database_record(record=entry) for entry in database_entries]
+        except Exception as e:
+            raise Exception(f'Error getting webhooks from database: {e}')
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)